### PR TITLE
docs: added documentation for AKS configuration and deployment

### PR DESCRIPTION
**Pull request by AngeloB-AIS**
#### PR Type:
- [x] New documentation
- [ ] Documentation Update
- [ ] Fix/Error Correction
- [ ] Typographical Error
- [ ] Other (Specify)

#### Detailed Description:
docs: added documentation for AKS configuration and deployment 

#### Checklist:
- [x] I have reviewed the contribution guidelines.
- [x] My documentation is in the correct directory/folder.
- [x] My changes do not duplicate existing documentation.
- [x] I have used spell check and my document is grammatically correct.
- [ ] If relevant, I have updated or added to the FAQs or Glossary sections.

### CODE REVIEW

File: guides/setup/aks.mdx [hunk 1]
```diff
@@ -1,4 +1,51 @@
...
+        * Leave defaults
+    * **Tags**
+        * Add tags if necessary, otherwise, leave defaults.
+    * **Review + Create**
+        * If there are validation errors that arise during the review, like a missed mandatory field, address the errors and create. If there are no validation errors, proceed to create.
+        * Wait for your deployment to complete before proceeding.
+4. **Connecting to your AKS Cluster**
+    * In the **Overview** section of your AKS cluster, there is a **Connect** button at the top. Choose whichever method suits you best and follow the on-screen instructions. Make sure to run at least one of the test commands to verify that your kubectl commands are being run against your new AKS cluster.
+5. **Deploying Multiwoven**
+    * Please refer to the **Helm Charts** section of our guide to proceed with your installation of Multiwoven!
```
Comment (subintp) [inline guides/setup/aks.mdx:50]: hyperlink helm chart link

File: guides/setup/aks.mdx [hunk 2]
```diff
@@ -1,4 +1,51 @@
...
+
+* An active Azure subscription
+* Basic knowledge of Kuberenetes and Helm
+
+**Note:** AKS clusters are not free. Please refer to https://azure.microsoft.com/en-us/pricing/details/kubernetes-service/#pricing for current pricing information.
+
+
+**1. AKS Cluster Deployment:**
+1. **Select a Resource Group for your deployment:** 
+    * Navigate to your Azure subscription and select a Resource Group or, if necessary, start by creating a new Resource Group.
```
Comment (subintp) [inline guides/setup/aks.mdx:20]: Add screenshots for every steps

**Review by subintp — CHANGES_REQUESTED** [review]

File: guides/setup/aks.mdx (hunk 2 above)
Comment (AngeloB-AIS) [reply guides/setup/aks.mdx:20]: @subintp done

File: guides/setup/aks.mdx (hunk 1 above)
Comment (AngeloB-AIS) [reply guides/setup/aks.mdx]: @subintp done

**Comment by AngeloB-AIS** [comment]
@subintp , I didn't realize the structure of main had changed so dramatically since I created this PR. I'm going to cancel this PR, create a new branch off of main and add my stuff there. I've saved the contents of the file, I just need to move it.

**Pull request closed without merging**